Clients/GUI/Startup.cs

```diff
@@ -32,6 +33,10 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<AuthStateProvider>();
             services.AddScoped<AuthenticationStateProvider>(
                     provider => provider.GetRequiredService<AuthStateProvider>());
+            services.AddServerSideBlazor().AddHubOptions(o =>
```

Comment:
This breaks sign in, so it says that you are not signed in when getting to user info after signing in

Reply:
fixed